src/main/java/org/openstreetmap/atlas/geography/atlas/raw/creation/OsmPbfReader.java

```diff
@@ -526,8 +526,7 @@ private void processWay(final Entity entity)
      * However, if we filter out a PBF {@link Way}, we want to keep track of all {@link Node}s that
      * make up that {@link Way}, in case we need to remove them as well. We are going to keep track
      * of all filtered PBF {@link Node}/Atlas {@link Point} identifiers, and use subAtlas
-     * functionality to filter them out after the Atlas is built. For now, ignore Relations. TODO
-     * Come back to Relations!
+     * functionality to filter them out after the Atlas is built. For now, ignore Relations.
```

Comment:
Having removed the TODO would indicate that it is not required anymore, however you still have "For now..." which indicates that you plan on introducing it later. Is the plan still to introduce code to not ignore Relations in the future, if not then we should just update the comments to read "This ignores relations" or something to that effect.

Reply:
Good callout! The intent here is to filter out any OSM Node/Atlas Point identifiers that make up the filtered Way. If we were to start removing Nodes from Filtered Relations - that logic may get very tricky, since the filtered Relation may not use the Node, but some underlying way might. We want to avoid this, and push this logic to way-sectioning code. I've updated the comment.